mmseg/apis/train.py

```diff
@@ -100,20 +101,23 @@ def train_segmentor(model,
     data_loaders = [build_dataloader(ds, **train_loader_cfg) for ds in dataset]
 
     # put model on gpus
```

Comment:
I think this should be revised as
```suggestion
    # put model on devices
```